package.json

```diff
@@ -320,10 +319,10 @@
         "uniqid": "^5.2.0",
         "uuid": "^8.3.2",
         "wait-on": "^3.3.0",
-        "webpack": "^4.47.0",
-        "webpack-bundle-analyzer": "^3.6.0",
-        "webpack-cli": "^3.3.10",
-        "webpack-dev-server": "^3.10.1",
+        "webpack": "^5.92.0",
```

Comment:
from the screenshot, the elements are almost doubling in size. was that expected? iirc dev console assets sizes after the webpack 4 upgrade were reduced so i'm curious what's happening here